ext/gd/gd.c

```diff
@@ -3633,6 +3633,11 @@ PHP_FUNCTION(imagescale)
 
 	im = php_gd_libgdimageptr_from_zval_p(IM);
 
+	if (tmp_h < 0 && tmp_w < 0) {
+		zend_value_error("$(width) and $(height) cannot be both negative");
```

Comment:
I don't think the parens make sense here:
```suggestion
		zend_value_error("$width and $height cannot be both negative");
```